### DISCUSSION TITLE
module returns the error "No related information"

### DISCUSSION THREAD

**Discussion by monebo90** [YetiForceCompany/YetiForceCRM#16674] [How to]
After switching server the relationship module returns error "No related information"
I saw that there was an update, but after doing it the error remains.
Can anyone tell me what's wrong?

**Comment by rskrzypczak** [comment]
You probably changed the server incorrectly.
The database and file may be inconsistent.
It's hard to be specific here, because who knows what's going on here.
However, check if the versions match:
- database (vtiger_version)
- files: config/version.php

**Reply by monebo90** [reply]
This is what I get
	'appVersion' => '6.4.0',
	'patchVersion' => '2022.08.25',
	'lib_roundcube' => '0.3.1',
